X/Xorg_xextproto/build_tarballs.jl

```diff
@@ -5,7 +5,7 @@ using BinaryBuilder
 name = "Xorg_xextproto"
 version = v"7.3.0"
 # We bumped the version number because we built for riscv64
-ygg_version = v"7.3.1"
+ygg_version = v"7.3.2"
```

Comment:
Doesn't seem necessary?

Reply:
Version `7.3.1+0` has been built and is available. If you say that a switch to `AnyPackage` is possible for a new version `7.3.1+1` then I can undo that bump.

Reply:
Yes.  No compat bounds will change and in practice nothing else should visible change, besides the fact the artifact would be able for all platforms (but for those already existing literally nothing should change)

Reply:
Done.